src/state/slices/opportunitiesSlice/thunks.ts

```diff
@@ -3,25 +3,22 @@ import type { AccountId } from '@shapeshiftoss/caip'
 import { DefiProvider, DefiType } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
 import { store } from 'state/store'
 
-import { foxEthLpAssetIds, foxEthStakingIds } from '../opportunitiesSlice/constants'
+import { foxEthStakingIds } from '../opportunitiesSlice/constants'
 import { opportunitiesApi } from '../opportunitiesSlice/opportunitiesSlice'
 
 export const fetchAllLpOpportunitiesMetadata = async (options?: StartQueryActionCreatorOptions) => {
-  const { getOpportunityMetadata, getOpportunitiesMetadata } = opportunitiesApi.endpoints
+  const { getOpportunitiesMetadata } = opportunitiesApi.endpoints
 
   await Promise.allSettled([
-    ...foxEthLpAssetIds.map(opportunityId =>
-      store.dispatch(
-        getOpportunityMetadata.initiate(
-          {
-            opportunityId,
-            opportunityType: DefiType.LiquidityPool,
-            defiType: DefiType.LiquidityPool,
-            defiProvider: DefiProvider.UniV2,
-          },
-          // Any previous query without portfolio loaded will be rejected, the first successful one will be cached
-          { forceRefetch: false, ...options },
-        ),
+    store.dispatch(
```

Comment:
can just do a single dispatch here, no `allSettled` required now

Reply:
Still required to dispatch these two?

```typescript
    store.dispatch(
      opportunitiesApi.endpoints.getOpportunitiesMetadata.initiate(
        {
          defiType: DefiType.LiquidityPool,
          defiProvider: DefiProvider.UniV2,
          opportunityType: DefiType.LiquidityPool,
        },
        // Any previous query without portfolio loaded will be rejected, the first successful one will be cached
        { forceRefetch: false, ...options },
      ),
    ),
    store.dispatch(
      getOpportunitiesMetadata.initiate(
        {
          opportunityType: DefiType.LiquidityPool,
          defiType: DefiType.LiquidityPool,
          defiProvider: DefiProvider.OsmosisLp,
        },
        { forceRefetch: false, ...options },
      ),
    ),
```

Reply:
you should probably disregard everything i've reviewed on this PR at this point, derp.